APISpecification.yaml

```diff
@@ -235,9 +235,54 @@ paths:
                       $ref: '#/components/schemas/PopularSpecies'
         '500':
           $ref: '#/components/responses/500InternalServerError'
+  /api/metadata/genome/{genome_id}/dataset/{dataset_type}/attributes:
+    get:
+      description: Returns a list of attributes for a genome.
+      parameters:
+        - name: genome_id
+          in: path
+          required: true
+          schema:
+            type: string
+            default: a7335667-93e7-11ec-a39d-005056b38ce3
+        - name: dataset_type
+          in: path
+          required: true
+          schema:
+            type: string
+            default: homologies
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  release_version:
+                    type: string
+                    example: "110.1"
+                  attributes:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Attributes'
+
+        '404':
+          description: Genome not found
```

Comment:
Technically, this response could be caused by 2 reasons: either genome id is incorrect, or the dataset type is incorrect. You may want to incorporate that into the error message on line 279.